e2e: updated setup and teardown

yevh-berdnyk opened the pull request:
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number

*otherwise*

fixes #...

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
...

<!-- (Optional, remove if no changes to documentation) -->
Documentation change PR (review please): https://github.com/status-im/status.im/pull/xxx

### Review notes
<!-- (Optional. Specify if something in particular should be looked at, or ignored, during review) -->

### Testing notes
<!-- (Optional) -->

#### Platforms
<!-- (Optional. Specify which platforms should be tested) -->

- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted
<!-- (Optional. Specify if some specific areas has to be tested, for example 1-1 chats) -->

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test
<!-- (Specify exact steps to test if there are such) -->

- Open Status
- ...
- Step 3, etc.

<!-- (PRs will only be accepted if squashed into single commit.) -->

status: ready <!-- Can be ready or wip -->

briansztamfater committed:
Improve mention suggestions list

Signed-off-by: Brian Sztamfater <brian@status.im>

jinhojang6 committed:
Update translations

yevh-berdnyk committed:
e2e: updated setup and teardown

yevh-berdnyk committed:
e2e: updated setup and teardown

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 08239666 | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-12789/1/) | 2021-11-04 05:31:57 | ~14 min | `android` | [:package:`apk`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-211104-051730-082396-pr12789-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-211104-051730-082396-pr12789-universal.apk)|

cammellos committed:
Add confirmation of p2p messages

Signed-off-by: Andrea Maria Piana <andrea.maria.piana@gmail.com>

snyk-bot committed:
fix: test/appium/requirements.txt to reduce vulnerabilities

The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-AIOHTTP-1584144

Signed-off-by: Churikova Tetiana <churikova.tm@gmail.com>

yevh-berdnyk committed:
update base_test_case.py

yevh-berdnyk committed:
a

flexsurfer committed:
[#11895] No last message when opening chat but it was visible on chat-preview

Signed-off-by: andrey <motor4ik@gmail.com>

churik committed:
...

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2373622">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/50dd1e745c4b4dc0954d5fd8d82ae1a5?auth=754e2fbec95f913b4e30845cc8ff13a1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/3047d82b1d63416f9265ea1196ac74ea?auth=68e3fee198242bf0647ee81e2f612c04">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2373623">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9dd97c141df44fe9a0d6001dc787d6d6?auth=89ef96e65cb46380fee34f1c6125cd35">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d99af559cd68407bad9479bc48a5bf92?auth=ff0877ea24c1a196169888665abb8905">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik committed:
...

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2374894">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1ed4cdc932174a209fc4b96c7794fa39?auth=c8c3e179382ac683ded0ab8941a2617b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7779f2fcb82e45f5a79cb3ded7dd5271?auth=2c6e36b15417ca62eb1e38c3e22c093d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2374893">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7e5e69c58f78443fad26f2fcad16c11f?auth=03a65daf69ed52836ddc8137c6c22dfc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7246c0a01f6a4f6dacb8207668d7f84a?auth=0a73edfee9dc6db620790906c4fd92c6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik committed:
exp2

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2374919">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0f0f72a684124c0193f11fa11dfc5388?auth=d4e853e487dd04537d53d76866248bc5">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/11ca22908c8c4ffdb6be0fc49750bdc8?auth=95b451fb67bc53e4561a409ea66a4bd4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2374904">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/753d768813ff4b54a20fa0df23b2aa53?auth=94cf1406289e0fbfeb51abdb8cdcc405">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7ff7d47bb0e448aaa67f80699198cbcf?auth=8e93d971de3410d7f66e00807d602dcf">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

yevh-berdnyk committed:
temp

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2375213">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c79c9f0f76154f0cadc83a3db520025b?auth=7521371fedf281cf1ef8fe5d046dd8c9">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f56d606d92074018bdd3399cce830f49?auth=4a2dfad4b7e678ddb77ff93494b68299">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2375198">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cbaf9910d63847428a5d0334cea968ff?auth=c37056b418fe5e060f3bba4a0d29916d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5860014f12a64632931db53d4520bf8d?auth=30e8fb630a2016c03026b77dfe6cdbf8">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

yevh-berdnyk committed:
temp2

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2375468">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f5c38410a41640eabcf756adb1337830?auth=f0aaf3e58ec0a0fd7c18cc40b210c11f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6d0a595d2f1a43cfbd95054d5c7eb816?auth=8be9bb5c8ca837c725a2e8836836b866">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2375453">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/73f592fa0f6b4c9f8c82ad1a38e5d576?auth=fbf4bf462ab930aba2b57b0fef13f590">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b4368de42f0c4732879b6890be472f54?auth=7f5d69f4c2abc71e75546718a147aade">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

yevh-berdnyk committed:
conftest

bitgamma committed:
sync activity center

Signed-off-by: Michele Balistreri <michele@bitgamma.com>

flexsurfer committed:
[#12858] [ANDROID] Changing screen orientation results in breaking images / layout

Signed-off-by: andrey <motor4ik@gmail.com>

rasom committed:
[#12848] Add an option to submit error report to GH

churik committed:
e2e: dependencies

Signed-off-by: Churikova Tetiana <churikova.tm@gmail.com>

rasom committed:
[#12782] Re-open app on the last open chat

cammellos committed:
[Fixes: #12869] Save contact on unblock

Parveshdhull committed:
Decline pending group invites from user, when user is blocked (#12878)

churik committed:
e2e: small fixes

Signed-off-by: Churikova Tetiana <churikova.tm@gmail.com>

rasom committed:
Allow sending messages offline (non-datasync messages)

rasom committed:
[#12813] Remove POA network from the list

bitgamma committed:
sync leftovers

Signed-off-by: Michele Balistreri <michele@bitgamma.com>

shivekkhurana committed:
Bump release to 1.18.0

Signed-off-by: Shivek Khurana <shivek@status.im>

shivekkhurana committed:
Add 1.18 to RELEASES.MD

Signed-off-by: Shivek Khurana <shivek@status.im>

bitgamma committed:
fix pairing for new cards

Signed-off-by: Michele Balistreri <michele@bitgamma.com>

cammellos committed:
Fix syncing public chats

rasom committed:
Fix unique :key warning on add pubchat screen

rasom committed:
[#12898] Fix endless loading in pubchats

rasom committed:
[#12907] Fix error on 'Accept and join' in referral chat

dependabot[bot] committed:
Bump lxml from 4.6.2 to 4.6.5 in /test/appium

Bumps [lxml](https://github.com/lxml/lxml) from 4.6.2 to 4.6.5.
- [Release notes](https://github.com/lxml/lxml/releases)
- [Changelog](https://github.com/lxml/lxml/blob/master/CHANGES.txt)
- [Commits](https://github.com/lxml/lxml/compare/lxml-4.6.2...lxml-4.6.5)

---
updated-dependencies:
- dependency-name: lxml
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

yevh-berdnyk committed:
updated xdist

yevh-berdnyk committed:
updated pytest version

yevh-berdnyk committed:
test

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390029">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/665b15bbef444daaa273e6147eb7a638?auth=4c7710c256a5dd67e0341dc7f2077a0c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/41a2be18becb4c10a0178389ca0cd275?auth=18d05a2629d214ecb7de85e37da33f65">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390044">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/665b15bbef444daaa273e6147eb7a638?auth=4c7710c256a5dd67e0341dc7f2077a0c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/41a2be18becb4c10a0178389ca0cd275?auth=18d05a2629d214ecb7de85e37da33f65">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

yevh-berdnyk committed:
base test case

yevh-berdnyk committed:
base test case

yevh-berdnyk committed:
base test case loop

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390539">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1d0959f4c21747df84acaa49f2a9fb85?auth=ecf56702381fee8cec6109742a17b8a0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/36848725b1794af19272831a00d93a22?auth=9eaf75e700548b9ed03c3b83b2d7c422">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390554">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1d0959f4c21747df84acaa49f2a9fb85?auth=ecf56702381fee8cec6109742a17b8a0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/36848725b1794af19272831a00d93a22?auth=9eaf75e700548b9ed03c3b83b2d7c422">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390786">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fc9399b617134f6bb86177864def65c7?auth=0217ace3a0b45a56d56d764bfbf6952a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/53b1cf3c07944a73916dc4991f57d92c?auth=8151da5985b5a62775ace42ede9e7485">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390787">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fc9399b617134f6bb86177864def65c7?auth=0217ace3a0b45a56d56d764bfbf6952a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/53b1cf3c07944a73916dc4991f57d92c?auth=8151da5985b5a62775ace42ede9e7485">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 3
Failed tests: 0
Passed tests: 3
```
<h3>Passed tests (3)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390789">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/11c31cfddda04b24a6735520a26a58c6?auth=b4b1aea04690a2d70709f1087f745251">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8726d3aa9eb248f193e7d04f25bd356a?auth=837f19bfd7015280a14836a2f5717917">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390790">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/11c31cfddda04b24a6735520a26a58c6?auth=b4b1aea04690a2d70709f1087f745251">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8726d3aa9eb248f193e7d04f25bd356a?auth=837f19bfd7015280a14836a2f5717917">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2390791">test_mark_all_messages_as_read_public_chat</a>, id: 6270 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/92be43fb15da45c190023989a8b4e1b2?auth=3683155390658da1aae44f47aa5e95b4">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/67871f24cba740a580657c1fa4f8039d?auth=0e8adc636dff3cf28a633feed3b1e6f9">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

yevh-berdnyk committed:
cleanup

status-im-auto commented:
## 88% of end-end tests have passed
```
Total executed tests: 69
Failed tests: 8
Passed tests: 61
```
```
IDs of failed tests: 5387,5310,5762,6228,6253,5346,6249,6240 
```
<h3>Failed tests (8)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&APK_NAME=https://status-im-nightlies.ams3.digitaloceanspaces.com/StatusIm-Mobile-211214-055900-81b60c-nightly-x86.apk&PR_ID=12789&TR_CASE_IDS=5387,5310,5762,6228,6253,5346,6249,6240">Rerun tests</a></li><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391435">test_delete_chats_via_delete_button_rejoin</a>, id: 5387 </b></td></tr><tr><td><p><blockquote><div>## Public chat 'public-delete-long-press' is created successfully!</div><div>Device 1: Looking for a message by text: test message:novawhtqga </div></blockquote></p><code> Messages are not fetched when rejoining public chat after deleting</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/39aff64daa7e4fc0afc64c62f874bc74?auth=9543a678171773976db2326c6ad38055">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/39aff64daa7e4fc0afc64c62f874bc74/0059screenshot.png?auth=9543a678171773976db2326c6ad38055">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391415">test_offline_is_shown_messaging_1_1_chat_sent_delivered</a>, id: 5310 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for chat: 'Closed Lumbering Hind' </div><div>Device 1: Wait for element 'Button' for max 30s and click when it is available </div></blockquote></p><code> Device 1: Button by accessibility id:`notifications-unread-badge` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e84591d084947568d0b497def6d68ce?auth=117659a846521e44a20fcfe877e38288">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/0e84591d084947568d0b497def6d68ce/0019screenshot.png?auth=117659a846521e44a20fcfe877e38288">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/1888120bd5674e388bbdbed2a065ab20?auth=700207297771b310d9aba9d2eeec6873">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/1888120bd5674e388bbdbed2a065ab20/0022screenshot.png?auth=700207297771b310d9aba9d2eeec6873">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391447">test_pair_devices_sync_one_to_one_contacts_nicknames_public_chat</a>, id: 5762 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find 'Button' by 'xpath': `//*[@text="Delete" or @text="DELETE"]` </div><div>Device 1: Tap on found: Button </div></blockquote></p><code> Message I am blocked user from previously blocked user is fetched</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c360707ec2464f2cacaffb331a5a003f?auth=e16f4e0a87b57404dd126dba7fcf3ecd">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/c360707ec2464f2cacaffb331a5a003f/0110screenshot.png?auth=e16f4e0a87b57404dd126dba7fcf3ecd">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9f855f57aaf64a408cde1bbcfc9fc54c?auth=c85a8ba5aeff229a016935ee02fb63ac">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/9f855f57aaf64a408cde1bbcfc9fc54c/0038screenshot.png?auth=c85a8ba5aeff229a016935ee02fb63ac">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/4023dc7cde324d19a2123178e6eb7e0e?auth=f5aee6770815919a927abb90194465a2">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/4023dc7cde324d19a2123178e6eb7e0e/0017screenshot.png?auth=f5aee6770815919a927abb90194465a2">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391455">test_mobile_data_usage_complex_settings</a>, id: 6228 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find 'Button' by 'xpath': `//*[@text="Use mobile data"]/following-sibling::android.widget.Switch[1]` </div><div>Device 1: Find 'Button' by 'xpath': `//*[@text="Ask me when on mobile network"]/following-sibling::android.widget.Switch[1]` </div></blockquote></p><code> Not connected to history node after enabling fetching on mobile data
     Chat history was not fetched with mobile data fetching ON</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/801a70c36c2d4028b32eefc4faa1492e?auth=330a0473b56e230ae50abe8b95ef95bc">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/801a70c36c2d4028b32eefc4faa1492e/0029screenshot.png?auth=330a0473b56e230ae50abe8b95ef95bc">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f9dd06212f1f4627b60e43151cc48fdf?auth=e8dc920538effb64add939296206ac55">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/f9dd06212f1f4627b60e43151cc48fdf/0028screenshot.png?auth=e8dc920538effb64add939296206ac55">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391461">test_send_eth_in_1_1_chat_transaction_push</a>, id: 6253 </b></td></tr><tr><td><p><blockquote><div>Transaction #5, amount is 0.001365421</div><div>{'blockNumber': '11609753', 'timeStamp': '1639378507', 'hash': '0x4cc2f6239c95e0761966f2f59e3e15d5dd6260940980ad785cba062807b8aefe', 'nonce': '2538', 'blockHash': '0xd29d05bf234d8e4d45bbb321c8b702ee9a9bdf844a58e5639610c68024c341f3', 'transactionIndex': '14', 'from': '0xf184747445c3b85ceb147dfb136067cb93d95f1d', 'to': '0xefca4fd47b98b9f01c4eef4827e576a01459728a', 'value': '1365421000000000', 'gas': '21000', 'gasPrice': '2500000000', 'isError': '0', 'txreceipt_status': '1', 'input': '0x', 'contractAddress': '', 'cumulativeGasUsed': '1121210', 'gasUsed': '21000', 'confirmations': '9057'}</div></blockquote></p><code> Transaction with amount 0.0014152913 is not found in list of ETH transactions, address is efca4fd47b98b9f01c4eef4827e576a01459728a during 300s</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c6b0e4f031764cbe9c26e40e640dd951?auth=d3f37daa02848e62b7214b249ff05e6c">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/c6b0e4f031764cbe9c26e40e640dd951/0041screenshot.png?auth=d3f37daa02848e62b7214b249ff05e6c">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/ede6e8f0ccc749bfa4c0e5a15af9919d?auth=323167040d3d003ceeca8fe5583aca5e">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/ede6e8f0ccc749bfa4c0e5a15af9919d/0026screenshot.png?auth=323167040d3d003ceeca8fe5583aca5e">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391426">test_collectible_from_wallet</a>, id: 5346 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find 'ProfileButton' by 'xpath': `//*[contains(@content-desc,'5 out of 5')]` </div><div>Device 1: Find 'ProfilePictureElement' by 'accessibility id': `chat-icon` </div></blockquote></p><code> Collectible image is not set as profile image</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c51c3bfb2cdf4bd4926c8af35b616f18?auth=980881fa6034ba7a71291b08bd16c303">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/c51c3bfb2cdf4bd4926c8af35b616f18/0051screenshot.png?auth=980881fa6034ba7a71291b08bd16c303">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391459">test_keycard_request_stt_from_daap</a>, id: 6249 </b></td></tr><tr><td><p><blockquote><div>Device 1: Enter default pin 111111 </div><div>Device 1: Enter default pin 111111 </div></blockquote></p><code> Device 1: Button by accessibility id: `maybe-later` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/05c78b9b77a94c10ba630daa3822f3e8?auth=9b623d2cd7e254815a4fcc9c3907b9fb">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/05c78b9b77a94c10ba630daa3822f3e8/0025screenshot.png?auth=9b623d2cd7e254815a4fcc9c3907b9fb">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391457">test_restore_account_from_mnemonic_to_keycard</a>, id: 6240 </b></td></tr><tr><td><p><blockquote><div>Device 1: Enter default pin 111111 </div><div>Device 1: Enter default pin 111111 </div></blockquote></p><code> Device 1: Button by accessibility id: `maybe-later` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a515c7ceb3134c58af03dfd18222f686?auth=16002eb1d23b29887fd94d12e983bc5e">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/a515c7ceb3134c58af03dfd18222f686/0025screenshot.png?auth=16002eb1d23b29887fd94d12e983bc5e">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (61)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391471">test_image_in_one_to_one_send_save_reply_timeline</a>, id: 6305 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a461d2fe3b5446a6a0bae9e58b1766ca?auth=4cedb9f8e80cf3994b2d8da29fd5945d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/a37b1c72e0d24ec2bac4a21d3954978c?auth=0b28716186392de255c0329164e9e856">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391453">test_add_account_to_multiaccount_instance_generate_new</a>, id: 6224 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d4b13506185c4978bec746ef8af4d589?auth=eb8ddaf715b004de74bcdac9f4a6612d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391442">test_can_add_existing_ens_on_mainnet</a>, id: 5502 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9dd2a45ad21341369989c23b846d7086?auth=25117ed1dfb7bfc33c55a13c13edc4e5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391414">test_request_stt_from_dapp</a>, id: 5309 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0c28913e3aab4bc9962de14e4a62e752?auth=1926b82d5291c812f6de68a8501a7e86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391468">test_keycard_can_see_all_transactions_in_history</a>, id: 6291 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/683132bb3d414f01aa4288410c65eb51?auth=e8349bef299452911ddfa927b4f3912d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391425">test_sign_message_and_2tx_in_batch_and_transactions_filters_from_daap</a>, id: 5342 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/166b466435d94a49aa75bedb74bfe276?auth=d462adfa75c48402880df7d736712ff1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391458">test_add_and_delete_watch_only_account_to_multiaccount_instance</a>, id: 6244 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b2632eb73cd24da48240def45299a38d?auth=f7d01df3f032603c9545a4ab03b30f24">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391452">test_open_blocked_secure_not_secure_inlalid_offline_urls</a>, id: 6210 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/108ac50357394e64964bd47884062b41?auth=84777c15d6d5ffbc428edd7121f86cc2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391438">test_open_public_chat_using_deep_link</a>, id: 5396 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9b372aa548874da29276a7ac20b8e69b?auth=f3f4076c51011445bfc4da864e326cd9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391411">test_offline_add_new_group_chat_member</a>, id: 3998 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/da57ad4977654a7a9f44e5b06dede1bd?auth=09fd4c7131b025642be4029e4bc9cb12">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/ce8fe74a6e1142c3b1fd34741ed5b40f?auth=19ef0d4a34083cc12d22f7adcec4de3f">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/de58562d5e9b491f8840ca60a97e2b2c?auth=8136e578cdde403c1fdc59a15eef283f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391418">test_send_non_english_message_to_newly_added_contact_on_different_networks</a>, id: 5315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c29be01d83b54b6faf6fe3ff7fee68b0?auth=d8a284f7a6cd9328dbb4e8180fcf0590">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8cf245e3db304668853bcb388b4bea55?auth=fc32faccd103b384bffa6f4dc8ae07ec">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391417">test_can_see_balance_and_all_transactions_history_on_cellular</a>, id: 5314 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/57098596fede41d7ab7e9af6f5726e60?auth=9101b43f6e0091c61ac62e2d7481ac7d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391440">test_insufficient_funds_wallet_positive_balance</a>, id: 5412 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dea9cf1cfcdb410ab30596445486c5b4?auth=e6fdc18c5b350faf1289851b7af66fc2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391436">test_delete_close_all_tabs</a>, id: 5390 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b9f8890531354344803d91a2c796c6cd?auth=199482d01011d2ec365180ac79ccc26f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391410">test_create_new_group_chat_messaging_pn_delivered</a>, id: 3994 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1b3c720059744db6b7ab55ef9be21dfa?auth=1c4455c7a3c77144ab22b57b1f8c9a23">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/ec0fb3ae9c234e6a851b8f9a8c06f465?auth=93c00375db8af49376bc451776f6451b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391427">test_send_token_with_7_decimals</a>, id: 5350 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4c56807f384c48b98eb17b4fed8ed988?auth=26a974c3c5c14d7028db48495a7fe8f2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391474">test_send_transaction_set_recipient_options</a>, id: 6328 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7f99e10c205c457bb93d227ce3c931a5?auth=f9b4f6c89a128412db374d537225ad99">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391460">test_keycard_sign_message_and_transactions_from_daap</a>, id: 6251 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/118d6189abf54109acd21afc104f5cf1?auth=017ef1d34b687f1ceb858f95dbc4ade5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391470">test_recover_account_from_new_user_seedphrase</a>, id: 6296 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e29abfe5526240688c4bd1df35c47954?auth=0f2eaac322f8b8b10a082107bb048ad8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391446">test_keycard_can_recover_keycard_account_card_pairing</a>, id: 5758 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5116e14db8334d7eb79b659fd2c9a0ba?auth=e5927ba57cfb265a83e475baac28635f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391450">test_block_user_from_public_chat</a>, id: 5786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e363e7069bb840ba92fc0a6b5aa795f0?auth=991ee68b5499d49b289588523f611099">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/162fce17421443d5b17fe05061be6c32?auth=ed3a01f0131ff5e70c07a9a1e1c527f6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391473">test_send_audio_message_with_push_notification_check</a>, id: 6316 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/06c3ff640d0d492ca9ee161cebd2fddb?auth=5fff70a77431d7acba65d481c9041ba5">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f41b6bf94a86439fab4eb2ff52342941?auth=2e430eb0c19e5dacbee7d105c20456a7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391443">test_redirect_to_public_chat_tapping_tag_message_fetch_more_history</a>, id: 5675 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/57471deb726f41528f0ae8d742c46d50?auth=f22fae08a371e0eb0d784505537d6e79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391476">test_restore_account_migrate_multiaccount_to_keycard</a>, id: 6645 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a7419000c4a74850b2a8d27014d645b1?auth=91e2432f26c0f2ed5219f69ba72d6e17">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391478">test_edit_delete_message_in_one_to_one_and_public_chats</a>, id: 695843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/edada27901d24ea6b88fa2727fb36084?auth=ceec146fc631030664918033b31adbc5">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6aef12d5c4b4424bbfefd227127c1e15?auth=b102c1b8e85299e216e7c4f52e3d6a32">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391454">test_ens_mentions_pn_and_nickname_in_public_and_1_1_chats</a>, id: 6226 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9195584b209b4d35a0f6958ca6257c70?auth=0ee045ebfcaade6e31e02a3d36322ae0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8940133dc960424e8a1b629bce38d2be?auth=1c30a5b9c1987bf22ffadc0a90406796">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391467">test_keycard_send_eth_from_wallet_to_address</a>, id: 6289 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/429cf9fe7da24a0daad75440a70c2fe9?auth=09e3d02104929d9aaf8c39d7477941c4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391444">test_keycard_create_login_restore_unlock_same_seed</a>, id: 5689 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e44149d69a60453a8ec56e18a9911944?auth=b22e19489d3eb194d5b89f5eadcb18c9">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7e761317dd874bc7847965f6d2f68633?auth=7e7c5e8df06792786f91937fb1932bb4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391464">test_add_account_to_wallet_private_key_and_seed_phrase</a>, id: 6272 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1fe649f933e346089847215a6f62c02c?auth=0160b29b0de318f45cc13158938949cd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391420">test_long_press_to_delete_chat</a>, id: 5319 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/85d1538fc43a41fdbd756cb9e04054ff?auth=67940e976d4fdfe6237d3979ad5a701f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391419">test_copy_and_paste_messages</a>, id: 5317 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/74d1359cac864df087610ff426da9ad6?auth=4a2eabc5002d88ec47980740248a6a6f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391434">test_open_transaction_on_etherscan_copy_tx_hash</a>, id: 5384 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2751b9f731334b9482490f16c5adc087?auth=88e976e1b4ae37bdc75ead586c2bbda9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391429">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f091336fe84f460593a98d5f36c5dbe8?auth=265f2f96d64a228f0e3b94fd0f594df2">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b346d454ef0b4284bec4738f5ea3f228?auth=056f3d9add015c075277379239110092">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391413">test_send_eth_from_wallet_to_address_incorrect_password</a>, id: 5308 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e44c9c0c994141648c31e0cad834f1df?auth=8c11684ea919359c4069a2a96f5cadd3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391439">test_start_chat_with_ens_mention_in_one_to_one</a>, id: 5403 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/48289c8de89f46a985174f7fb4c242b2?auth=7785459fe09223a02d6621fe064afd93">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391463">test_decline_transactions_in_1_1_chat_push_notification_changing_state</a>, id: 6265 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cf286e9e01f1483dbb6a6345b1632644?auth=5477b7a6cde28911be3cbe01fcd03a59">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/ad7bea2c43fa468e9b47a91e68eaf385?auth=063fc7673ddd4972900397fbaa1dbc85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391448">test_install_pack_and_send_sticker</a>, id: 5782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6d46f47273a5446c96d9a87bbec4d73f?auth=7f98b85cc199f1d6df3e74671ee66d6c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/c1b21a8e71bb440f9406199b93ba900a?auth=239fc7216dc52d2ffd417aafd9413f68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391428">test_switch_users_special_char_password_and_add_new_account_logcat</a>, id: 5356 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/138b6a27bcc440f8b089f735a8df8afc?auth=e2f838f00af7321f7f87967c34d41222">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391424">test_manage_assets</a>, id: 5341 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8a79b11bb16148c7a24da2a122e38185?auth=f62d3c7fd99b2cd3e14e9492f43252ac">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391475">test_browser_managing_bookmarks</a>, id: 6633 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/406268ef2fe0483fa00d007665e0bb56?auth=70034ef631ca966254b69abcc4f5dd15">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391412">test_open_chat_by_pasting_chat_key_check_invalid_chat_key_cases</a>, id: 5304 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6e6a2b1f2c4e47f3a0e99b3c27e98f2a?auth=73cd0a59d53585fb4384945237d684d7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391449">test_can_use_purchased_stickers_on_recovered_account</a>, id: 5783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6c7249afceb949c0b49f9a209dc74915?auth=2c00956fccaefe3beebbe584c9fc011d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391445">test_dapps_permissions</a>, id: 5738 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a5b6b44cbfd64183ab1dadc0e2075469?auth=dfa487abc312dddbd11ea253e14473b2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391437">test_account_recovery_with_uppercase_whitespaces_seed_phrase_special_char_passw_logcat</a>, id: 5394 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b44944fd2e7c4deb81651ce9945fb381?auth=f13b2c4c5bf275cef03f4016cb511a0d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391465">test_send_eth_to_ens_in_chat</a>, id: 6279 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fa477f96b676456489c186bcf652726e?auth=596e0f6b60141748b30e520c02eaa339">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/fe868a9f8831443ea338bd6bfe6ad00b?auth=5f0464c8ac6a2bedf4f999b2f353389d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391421">test_share_copy_contact_code_and_wallet_address</a>, id: 5323 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/97ac5206176f4f91897d7c91adf4cff7?auth=daa666f7026d4cf11e43e9a7c0bc39ce">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391423">test_wallet_set_up</a>, id: 5335 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b6cdca6b442548d5984d9d997d1c76ac?auth=c047fd9b90bdb01f778da96b6fbe4b99">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391451">test_send_transaction_with_custom_token</a>, id: 6208 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b536e68169934e43946de060422acbca?auth=fc76be4cf87e416c7b2931627e001a7d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391432">test_home_view</a>, id: 5379 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3f7a319cb34a4984be26f3ba86a8bedd?auth=fdb2d8290ec32e45c31ccd76124ef150">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391441">test_logcat_backup_recovery_phrase</a>, id: 5419 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dae13a55564e499e88c0b149c8b8e7c5?auth=84bd7750b775c5d3e84035a6d5b770a9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391462">test_request_and_receive_stt_in_1_1_chat_offline</a>, id: 6263 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/175cefb8f53a433e8500fffc47cc4cfd?auth=80f24f2cd1451055d6af1a1a2c7bac65">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/59dd801f61794b3195e50362a4d7d555?auth=a9069fd7a78f980deb5c945b451ccb3a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391477">test_set_profile_picture</a>, id: 6646 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9ec314a1cb264e5eb2e0d889d5951ac2?auth=7adcafc7be484335e150fb164eab87e9">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5743ce7772e842978e060a639500e296?auth=cbcee2c321ff817c3a6b2f6669899256">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391422">test_add_and_remove_mention_contact_with_nickname_from_public_chat</a>, id: 5332 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9153b54473ac453d8e5185c2a53bc5c1?auth=11de3bb50fbc7448a7ae9fc132be42fd">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/0c74c30b22654cf4ad6c4795df5738a6?auth=edd65d84ed062eeedafa00fecb9aabe2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391430">test_pass_phrase_validation</a>, id: 5363 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/88b749b123ae4c2eb701ae10c34e088f?auth=9854d8383cc343b8b0c0c9878c09e4a5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391433">test_user_can_see_collectibles_on_rinkeby_after_account_recovering</a>, id: 5381 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3fd027e2593a44099c659ae4a113a804?auth=8bc5a0ad00a9f169ad9985ca5ab5c29a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391469">test_keycard_send_eth_in_1_1_chat</a>, id: 6293 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/beb45a54e2644777852019f52d114182?auth=97196f69e75d3ffc422bb61e97fb4448">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/82985994d03e49baa5718b24303e17bc?auth=e90551f0399e60f7e1a556c0f0c12a57">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391431">test_send_and_open_links_with_previews</a>, id: 5373 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/09699890a8fc4325ad33c46c4fe13eab?auth=b50f48fddc40b95628f92d829b8bb7bc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/de6dad2b658c4c61835af94d86582ced?auth=cda18cbbc0f75aa0fab02b4af54ec440">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391472">test_reactions_to_message_in_chats</a>, id: 6315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/93c34a3a04774642af15b3a17c81b0ce?auth=63956308ebcfd0586b087c0c829dee29">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/0e5a64acb0164cc5809ea36b465c99a7?auth=ce41b1f84a61b3621c7ea850822af6be">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391456">test_fetching_balance_after_offline</a>, id: 6237 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c3a3e3c4b7bb444c8347ab347d7eeb24?auth=8800878f1e92499dcda5aef54eb5dcc9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391466">test_push_notification_1_1_chat_no_pn_activity_center</a>, id: 6283 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c7c1cc46e89c4bea9f3671a70fad5a99?auth=3b225dc4da013c9a1def3985007a8784">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/356cc007ad9b4105b3783d0d338a42c3?auth=9ebe931d0ac34deadbe8a1549df2f4e9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2391416">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f091336fe84f460593a98d5f36c5dbe8?auth=265f2f96d64a228f0e3b94fd0f594df2">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b346d454ef0b4284bec4738f5ea3f228?auth=056f3d9add015c075277379239110092">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik commented:
🥇

yevh-berdnyk committed:
Merge branch 'tests/new-setup-and-teardown' of github.com:status-im/status-react into tests/new-setup-and-teardown
